package.json

```diff
@@ -211,6 +212,7 @@
     "build-derefed-docs": "yarn install-api-docs && node api-docs/index.js api-docs/openapi.json",
     "watch-api-docs": "sane 'yarn build-derefed-docs' api-docs",
     "build-css": "NODE_ENV=production yarn webpack --config=config/webpack.css.config.js",
+    "build-charts-config": "NODE_ENV=production yarn webpack --config=config/webpack.chart-renderer.config.js",
```

Comment:
Not sure yet how this is going to get produced during deploys

Reply:
Would there be an option to run the docker container? I'd prefer to replicate similar conditions as prod locally.

Reply:
We can definitely have a dev-service that runs the docker container. Though I think we'll also have dev just produce images by shelling out to the binary.

Reply:
Could GCB create the images as master moves? Local environments could pull down the docker image and use it. For folks developing chart-renderer they could run the same container image with a volume mount.

Reply:
My plan right now was to have development just shell out to with node, since I've added it to `package.json`.

Reply:
> Could GCB create the images as master moves

It does this now. We still need to build the configuration file here though for it.